Project/Fall2020_CSC403_Project/FrmBattle.cs

```diff
@@ -86,37 +86,48 @@ public partial class FrmBattle : Form {
             {
                 enemy.OnAttack(enemy.Attacks[1].atkVal * -1);
             }
-        }
-        
-      UpdateHealthBars();
-      if (player.Health <= 0 || enemy.Health <= 0) {
-        instance = null;
-        Close();
-      }
-    }
-
-    // placeholder EventHandler function for the items button
-    private void btnItems_Click(object sender, EventArgs e)
+        }
+        
+      UpdateHealthBars();
+      if (player.Health <= 0 || enemy.Health <= 0) {
+        instance = null;
+        Close();
+      }
+
+      // adding random dialogue to the Battle using the textboxes
+      Random random1 = new Random();
+      Random random2 = new Random();
+      string[] playerResponses = new string[5] { "Take that", "I've got you now", "Ha. Weakling.", "You're going down", "What's the matter? Can't take a hit?" };
+      string[] enemyResponses = new string[5] { "Please, have mercy", "No. Stop.", "Ah", "Oh no", "How could you do this?" };
+
+      int playerResponse = random1.Next(0, 5);
+      int enemyReponse = random2.Next(0, 5);
+      textboxBattleEnemy.Text = enemyResponses[enemyReponse];      
+      textboxBattlePlayer.Text = playerResponses[playerResponse];
+    }

```

Comment:
In future revisions this should probably be its own function.  I.e. UpdateHealthBars() Say GenerateBattleResponses()